src/components/Modals/Receive/ReceiveInfo.tsx

```diff
@@ -18,40 +18,48 @@ import {
   useToast
 } from '@chakra-ui/react'
 import { utxoAccountParams } from '@shapeshiftoss/chain-adapters'
-import { Asset } from '@shapeshiftoss/types'
+import { Asset, UtxoAccountType } from '@shapeshiftoss/types'
+import last from 'lodash/last'
 import { useEffect, useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { useSelector } from 'react-redux'
-import { useHistory } from 'react-router-dom'
+import { RouteComponentProps, useHistory } from 'react-router-dom'
 import { Card } from 'components/Card/Card'
 import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
 import { QRCode } from 'components/QRCode/QRCode'
 import { Text } from 'components/Text'
 import { useChainAdapters } from 'context/ChainAdaptersProvider/ChainAdaptersProvider'
 import { useWallet } from 'context/WalletProvider/WalletProvider'
 import { ReduxState } from 'state/reducer'
+import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSlice'
 
 import { ReceiveRoutes } from './Receive'
 
 type ReceivePropsType = {
   asset: Asset
-}
+  accountId: AccountSpecifier
+} & RouteComponentProps
 
-export const ReceiveInfo = ({ asset }: ReceivePropsType) => {
-  const { chain, name, symbol } = asset
+export const ReceiveInfo = ({ asset, accountId }: ReceivePropsType) => {
   const { state } = useWallet()
   const [receiveAddress, setReceiveAddress] = useState<string>('')
   const [verified, setVerified] = useState<boolean | null>(null)
   const chainAdapterManager = useChainAdapters()
   const history = useHistory()
+  const { chain, name, symbol } = asset
 
   const { wallet } = state
   const chainAdapter = chainAdapterManager.byChain(chain)
 
-  const currentAccountType = useSelector(
+  let currentAccountType = useSelector(
```

Comment:
we wanna delete this selector so we can kill this preferences thing entirely

just initialize it to `''` and throw on line 62 if it's still empty to make the compiler happy

Reply:
I did that originally, but the throw broke the ethereum receive page. However, if I initialize as an empty string then typecast as UtxoAccountType it seems happy without the throw and without breaking ethereum. So I did that.